packages/protocol/lib/compatibility/version.ts

```diff
@@ -105,3 +124,57 @@ export class ContractVersionDelta {
     return deltas.map(d => d.toString()).join('.')
   }
 }
+
+/**
+ * A mapping {contract name => {@link ContractVersionDelta}}.
+ */
+export interface ContractVersionDeltaIndex {
+  [contract: string]: ContractVersionDelta
+}
+
+/**
+ * A mapping {contract name => {@link ContractVersion}}.
+ */
+export interface ContractVersionIndex {
+  [contract: string]: ContractVersion;
+}
+
+/**
+ * A mapping {contract name => {@link ContractVersion}}.
+ */
+export class ContractVersions {
+  static fromArtifacts = async (artifacts: BuildArtifacts): Promise<ContractVersions>=> {
```

Comment:
Add a doc to the method explaining that it will call the method `getContractVersion() (uint, uint, ... )` to retrieve the information, and what the default version will be if the method is not available

Reply:
Added docs to `getContractVersion`